README.md

```diff
@@ -54,38 +57,71 @@ Clone this repository to your desired folder.
 
 ## Install
 
-Install this project with:
+To install this project and its dependencies, follow these steps:
+
+[```bash
+npm install]
+
 
-## Usage
+### Usage
 
-To run the project, execute the following command:
+To use this project, create a new branch to add more features. However, to run the project, use the following commands:
+
+npm run build   # It will create a production-ready file
+npm start       # It will run the server on your machine
 
 ## Run tests
 
-To run tests, use the following command:
+This project includes linters or tests to avoid any kind of typos and ensure good practices. To run the linter tests on the HTML or any HTML file, use the following command:
+
+npx hint .
+
+To run the linter tests on stylesheets, use the following command:
+
+npx stylelint "*+/*.{css, scss)"
+
+To run the linter tests on JavaScript files, use the following command:
+
+npx eslint .
 
 ## Deployment
 
-You can deploy this project using:
+You can deploy this project using the deployment method of your choice.
 
 # 👥 Authors <a name="authors"></a>
 
-👤 **CVILLA09**
+👤 Cristian Villa
 
 - GitHub: [@CVILLA09](https://github.com/CVILLA09)
 
 # 🔭 Future Features <a name="future-features"></a>
 
 There are no planned future features for this project.
 
+# 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+# ⭐️ Show your support <a name="show-your-support"></a>
+
+Write a message to encourage readers to support your project
+
+If you like this project...
+
 # 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 We acknowledge Microverse and the coding partners at Microverse.
 
 # 📝 License <a name="license"></a>
 
-This project is under the [MIT License](LICENSE).
+Copyright 2023, [Cristian Villa]
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this ["Hello Microverse Project"] and associated documentation files, to deal in the ["Hello Microverse Project"] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the ["Hello Microverse Project"], and to permit persons to whom the ["Hello Microverse Project"] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the ["Hello Microverse Project"].
+
+THE ["Hello Microverse Project"] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE ["Hello Microverse Project"] OR THE USE OR OTHER DEALINGS IN THE ["Hello Microverse Project"].
```

Comment:
- [ ] Great job on incorporating the suggestions from the previous reviewer!  🎉🎉
- [ ] Instead of using the MIT license description, it would be more appropriate to include a direct link to the license. 
- [ ] This will provide the readers with accurate and comprehensive information about the MIT license and your readme would be more professional.✔️✔️✔️

Reply:
should i do this?
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository